app/components/ImageView.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Map, Marker, Popup, TileLayer, MapComponent } from 'react-leaflet';
+import L from 'leaflet';
+
+const IMG1 = 'https://raw.githubusercontent.com/cdeil/fermi-ts-maps/master/emin_0030_emax_0100.png';
+
+export default class Maps extends Component {
+  constructor() {
+    super();
+    this.url = 'https://raw.githubusercontent.com/cdeil/fermi-ts-maps/master/emin_0030_emax_0100.png';
```

Comment:
In this PR you've copied over the PNG images to this repo.
So I guess you should use the file from this repo here, then?